modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc

```diff
@@ -9,6 +9,11 @@ As a cluster administrator, you can assign an SR-IOV network interface to your V
 
 To do this, add the VRF configuration to the optional `metaPlugins` parameter of the `SriovNetwork` resource.
 
+[NOTE]
+====
+Applications which use VRFs need to bind to a specific device. The common usage is to use the `SO_BINDTODEVICE` option for a socket. `SO_BINDTODEVICE` binds the socket to a particular device as specified in the passed interface name, for example, `eth1`. In order to use `SO_BINDTODEVICE`, the application must have `CAP_NET_RAW` capabilities.
```

Comment:
Applications which use VRFs need to bind to a specific device. The common usage is to use the `SO_BINDTODEVICE` option for a socket. `SO_BINDTODEVICE` binds the socket to a device that is specified in the passed interface name, for example, `eth1`. To use `SO_BINDTODEVICE`, the application must have `CAP_NET_RAW` capabilities.

Reply:
A few nits

Reply:
fixed! :)